test/builtin/files_io/test_files.py

```diff
@@ -37,6 +37,16 @@ def test_get_and_put():
     check_evaluation(f"DeleteFile[{temp_filename}]", "Null")
 
 
+def test_get():
+    # Check that $InputFileName sets filename inside Get[]unction with Get[]
```

Comment:
This comment has several typos. Should be something like:
`# Check that $InputFileName is set inside Get[] function`